servicesContracts.py

```diff
@@ -1,10 +1,28 @@
 from typing import Any
-
-from artifacts import *
```

Comment:
You should be able to import all of the artifacts from the artifacts module. Python can parse JSON into `dict`-like objects.